base/bucket.go

```diff
@@ -34,7 +34,7 @@ const (
 	DefaultPool = "default"
 )
 
-const DefaultViewTimeoutSecs = 75 // 75s
+const DefaultViewTimeoutSecs uint32 = 75 // 75s
```

Comment:
Nit: take or leave: make this


```suggestion
const DefaultViewTimeoutSecs = 75 * time.Second
```

Then use as `DefaultViewTimeoutSecs.Milliseconds()` in return value?

Reply:
Still have to do the awkward uint32 conversion after converting from a duration to milliseconds though in real usage.

Given this is still old view code I'm not going to change it.